tests/components/litterrobot/conftest.py

```diff
@@ -35,3 +37,17 @@ def mock_hub(hass):
     hub.coordinator.last_update_success = True
     hub.account.robots = [create_mock_robot(hass)]
     return hub
+
+
+async def setup_hub(hass, mock_hub, platform_domain):
+    """Load a Litter-Robot platform with the provided hub."""
+    entry = MockConfigEntry(
+        domain=litterrobot.DOMAIN,
+        data=CONFIG[litterrobot.DOMAIN],
+    )
+    entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+
+    with patch.dict(hass.data, {litterrobot.DOMAIN: {entry.entry_id: mock_hub}}):
```

Comment:
Path the pypi package when you call `async_setup` instead of patching `hass` here (be sure to `await hass.async_block_till_done()` before leaving the patch context to ensure the patch remains in effect until setup is finished).

Reply:
Thanks, @bdraco. The tests definitely are not my strong suit... 😅